README.md

```diff
@@ -24,27 +24,28 @@ These goals will be accomplished within the following pipeline structure:
 ## Usage
 
 
-### Environment Set Up 
+### Environment Set Up
 
 Before running the repository (see details below), you need to perform the following steps:
 1. Install make if you have not already done so.
-2. Ensure you have access to [Slurm] (python train.py configs.config --experiment_name baseline_v1). 
+2. Ensure you have access to [Slurm] (python train.py configs.config --experiment_name baseline_v1).
 3. Create and initiate a cmap specific conda environment using the following steps:
-    1) Install miniconda:
+    1) Install micromamba:
     ```
-    mkdir -p ~/miniconda3
-    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh
-    bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
+    curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+    ./bin/micromamba shell init -s bash -r ~/micromamba
+    source ~/.bashrc
+    micromamba config append channels conda-forge
     ```
     2) Create environment:
     ```
-    conda create -y --name cmap python=3.10
-    conda activate cmap
-    conda install pytorch torchvision pytorch-cuda=12.1 -c pytorch -c nvidia
-    pip install -r /home/YOUR_USERNAME/2024-winter-cmap/requirements.txt
+    micromamba create -y --name cmap python=3.10
+    micromamba activate cmap
+    micromamba install -y pytorch torchvision pytorch-cuda=12.1 -c pytorch-nightly -c nvidia
```

Comment:
Even if pytorch nightly works in a simple test, let's avoid it since it will be less stable unless there is a specific need?

Reply:
I just retested with the stable version and it works! I will update the README with the modified code